queryset_sequence/__init__.py

```diff
@@ -331,6 +331,14 @@ def _fetch_all(self):
         if self._result_cache is None:
             self._result_cache = list(self._iterable_class(self))
 
+    @property
+    def _prefetch_related_lookups(self):
+        # for django ModelChoiceField, pending for removal
+        return any(map(
+            lambda qs: qs._prefetch_related_lookups,
+            self._querysets
+        ))
```

Comment:
This doesn't look right, `_prefetch_related_lookups` seems to be a tuple on `QuerySet`, not a boolean. If the goal is to just fix the `ModelChoiceField` but then the comment above this needs to be expanded.

Either way I'd like to see the comment describe what versions of Django this affects (and a reference to the Django ticket that will fix this).

Reply:
I updated the comment here and return a tuple, which...is still funky, but at least is the same type.